3.preprocessing_features/scripts/2.bulk_processing.py

```diff
@@ -22,6 +22,11 @@
 # In[2]:
 
 
+# Condition for how to normalize the plates
+normalize_with_U2OS = (
+    True  # Set to False if normalizing to whole plate versus just U2OS cell line
+)
+
```

Comment:
This helped!

Reply:
Thanks! :) I tried to make it clear and understandable for future reference haha